styles.css

```diff
@@ -0,0 +1,1016 @@
+/* =============== FONT =============== */
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap');
+
+/* #093b5f    dark-blue*/
+/* #8bc34a    green*/
+/* #ebebeb    light-grey*/
+
+/* =============== GLOBAL =============== */
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  overflow-x: hidden;
+}
+
+html {
+  font-size: 100%;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  color: #093b5f;
+}
+
+h2 {
+  padding: 2rem 0;
+}
+
+p {
+  color: #555;
+  line-height: 1.5;
+}
+
+.btn {
+  background-color: #8bc34a;
+  text-align: center;
+  display: inline-block;
+  padding: .5rem 2rem;
+  border-radius: 0.125rem;
+  text-decoration: none;
+  outline: none;
+  border: none;
+  font-size: 1rem;
+  color: #fff;
+  cursor: pointer;
+  transition: all .2s ease-in-out;
+}
+
+.btn:hover {
+  background-color: #6f9c3c;
+}
+
+.container {
+  max-width: 70%;
+  margin: 0 auto;
+}
+
+.home-wrapper {
+  display: flex;
+  padding-top: 3rem;
+}
+
+.left-container {
+  width: 80%;
+}
+
+.right-container {
+  width: 30%;
+  padding-left: 2rem;
+}
+
+.mobile-show {
+  display: none;
+}
+
+/* =============== HEADER =============== */
+
+header {
+  height: 25rem;
+  color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+header::before {
+  background: linear-gradient(to top, green, rgb(90, 90, 90) 50%);
+  z-index: -30;
```

Comment:
Can probably just combine both these header::before{
the z-index on line 106 will overwrite this.